fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/AddColumn.java

```diff
@@ -38,7 +42,19 @@ public AddColumn(String schemaName, String tableName, ColumnBase column) {
     @Override
     public void run(IDatabaseTranslator translator, Connection c) {
         String qname = DataDefinitionUtil.getQualifiedName(schemaName, tableName);
-        String ddl = "ALTER TABLE " + qname + " ADD COLUMN " + columnDef(column);
+
+        // DatabaseTypeAdapter is needed to find the correct data type for the column.
```

Comment:
I suggest putting this in CommonDatabaseAdapter as a common method

Reply:
because this is used by AddColumn only, so let's keep it here for now, if we can move whenever we need in the future.